webclient/core/repository_connector.py

```diff
@@ -1100,7 +1173,8 @@ def record_deletion(self):
             headers = {"Slug": self.record.ident_cely, "Content-Type": "text/turtle"}
             data = (
                 f"@prefix ore: <http://www.openarchives.org/ore/terms/> . "
-                f"<> ore:proxyFor <info:fedora/{settings.FEDORA_SERVER_NAME}/record/{self.record.ident_cely}>"
+                f"<> ore:proxyFor <info:fedora/{settings.FEDORA_SERVER_NAME}/record/{self.record.ident_cely}> ; "
+                f'<http://purl.org/dc/terms/creator> "{self.user}" .'
```

Comment:
Místo fixního užívání `http://purl.org/dc/terms/creator` bych definoval prefix `@prefix dcterms: <http://purl.org/dc/terms/> .` a pak referoval na `dcterms:creator`. Fedora s tím stejně takto pracuje, tak ať to nemate.